app/controllers/homepages_controller.rb

```diff
@@ -0,0 +1,16 @@
+class HomepagesController < ApplicationController
+  def index
+    @works = Work.all
+
```

Comment:
Hm, I see some interesting thoughts starting here. In your `homepage#index` view, you end up setting `spotlight_work = Work.spotlight_work(@works)`... what if you set `@spotlight_work = Work.spotlight_work(@works)` in this controller?